src/runtimes/node/list_imports.js

```diff
@@ -0,0 +1,50 @@
+const esbuild = require('@netlify/esbuild')
+const isBuiltinModule = require('is-builtin-module')
+const { tmpName } = require('tmp-promise')
+
+const { safeUnlink } = require('../../utils/fs')
+
+const getListImportsPlugin = ({ imports, path }) => ({
+  name: 'list-imports',
+  setup(build) {
+    build.onResolve({ filter: /.*/ }, (args) => {
+      const isImport = args.path !== path && !isBuiltinModule(args.path)
```

Comment:
`precinct` has the `includeCore` property to exclude built-in Node modules, which doesn't exist in esbuild. So we use `is-builtin-module` to that instead.